docs/release_guide.md

```diff
@@ -121,3 +123,28 @@ Make an announcement on the [mailing list](https://groups.google.com/g/openservi
 Skip this step if the release is a release candidate (RC).
 
 Open a pull request against the `main` branch making the same version updates as [above](#update-release-branch-with-patches-and-versioning-changes) so the latest release assets are referenced there.
+
+## Make version changes on docs.openservicemesh.io
+
+Associated version numbers need to be updated in the [docs.openservicemesh.io](https://github.com/openservicemesh/osm-docs/) repo.
+
+The files that need to be updated are:
+
+* [Readme](https://github.com/openservicemesh/osm-docs/blob/main/README.md)
+* [Manual Demo](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/getting_started/manual_demo.md)
+* [Health Probes](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/application_health_probes.md)
+* [Control Plane Health Probes](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/control_plane_health_probes.md)
+* [OSM MeshConfig](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/osm_mesh_config.md)
+* [Upgrade Guide](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/upgrade_guide.md)
+* [Installation](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/install/_index.md)
```

Comment:
Where is there a version here at all?

Reply:
It's the link of OSM Chart:

https://github.com/openservicemesh/osm-docs/blob/main/content/docs/install/_index.md#using-the-helm-cli

Reply:
Ah thanks, I see it now.